papers/hendrik_heuer/heuer.rst

```diff
@@ -0,0 +1,223 @@
+
+In the dimensionality reduction step, the 300-dimensional word vectors are projected down to a two-dimensional space so that they can be easily visualized in a 2D coordinate system (see Fig. 2).
+
+.. figure:: tsne_dimensionality_reduction.png
+
+   In the dimensionality reduction step, the word vectors are projected down to 2D :label:`egfig`
+
+For the implementation, the t-SNE implementation in scikit-learn is used:
+
+.. code-block:: python
+
+   from sklearn.manifold import TSNE
+
+   tsne = TSNE(n_components=2)
+   tsne.fit_transform( word_vectors )
+
+Visualization
+~~~~~~~~~~~~~
+
+After the dimensionality reduction, the vectors are exported to a JSON file. The vectors are visualized using the D3.js JavaScript data visualization library [Bos12]. Using D3.js, an interactive map was developed. With this map, the user can move around and zoom in and out.
+
+Results
+--------------
+
+The flow described in the previous section is applied to different revisions of Wikipedia articles. For this, a convenience sample of the most popular articles in 2013 from the English Wikipedia was used.  For each article, the last revision from the 31st of December 2013 and the most recent revision on the 26th of May 2015 were collected. The assumption was that popular articles will attract sufficient changes to be interesting to compare. The list of the most popular Wikipedia articles includes Facebook, Game of Thrones, the United States, and World War 2.
+
+The article on Game of Thrones was deemed especially illustrative for the task of comparing the topics in a text, as the storyline of the TV show developed between the two different snapshot dates as new characters were introduced. Other characters became less relevant and were removed from the article. The article on World War 2 was especially interesting as one of the motivations for the topic tool is to find subtle changes in data.
+
+Fig. 3 shows how different the global cluster, i.e. the full group of words on the maximum zoom setting, of the Wikipedia articles on the United States, Game of Thrones and World War 2 are.
+
+.. figure:: global_clusters.png
+
+   Global clusters of the Wikipedia articles on the United States (left), Game of Thrones (middle), and World War 2 (right). :label:`egfig`
+
+Fig. 4 shows four screenshots of the visualization of the Wikipedia articles on the United States including an overview and detail views that only show the intersection set of words, words only present in the 2013 revision of the article, and words only present in the 2015 revision of the article. 
+
+When applied to Game of Thrones, it is e.g. easy to visually compare characters names, i.e. first names, that were removed since 2013 and that were added in 2015. Using the online demo available [Heu15], this technique can be applied to the Wikipedia articles on the United States and World War 2. The technique can also be applied to visualize the Google search history of an individual.
+
```

Comment:
I added more on the analysis. Not entirely sure how to address the workflow question, but I tried providing more examples on how it could be used and what of the intention is.